app/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "simple-site",
+  "version": "0.0.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/codefellows/band-name-generator"
+  },
+  "description": "Example Site for JavaScript F2 at Code Fellows",
+  "dependencies": {
+    "express": "^4.12.4"
```

Comment:
When you install body-parser, if you want to save the module to your dependencies in the package.json, you do this:
npm install --save body-parser
(this will add body parser to the dependencies section)
or
npm install --save -dev body-parser
(this adds it to the dev dependencies section. In this case, you want it in the regular dependencies)